### ISSUE TITLE
Goombas in Super Mario 64 DS stages are tinted blue

### ISSUE THREAD

**Issue by GRAnimated** [magcius/noclip.website#457]
running on an i7 10700KF, RTX 2060. tested in microsoft edge and chrome
![image](https://user-images.githubusercontent.com/23510999/133951373-6023a331-0577-47ee-9c8c-3d530feaba22.png)

**Comment by magcius** [comment]
Yeah, the Goomba models seem to come with the color that Yoshi tints them when you spit them out. I don't know how the original game unsets the color.
